app/src/main/java/app/revanced/manager/ui/component/bundle/BaseBundleDialog.kt

```diff
@@ -159,22 +159,23 @@ fun BaseBundleDialog(
         )
     }
 
-    if (patchCount > 0) {
-        BundleListItem(
-            headlineText = stringResource(R.string.patches),
-            supportingText = if (patchCount == 0) stringResource(R.string.no_patches)
-            else stringResource(R.string.patches_available, patchCount),
-            trailingContent = {
-                if (patchCount > 0) {
-                    IconButton(onClick = onPatchesClick) {
-                        Icon(
-                            Icons.Outlined.ArrowRight,
-                            stringResource(R.string.patches)
-                        )
-                    }
-                }
+
+    BundleListItem(
+        headlineText = stringResource(R.string.patches),
+        supportingText = if (patchCount == 0) stringResource(R.string.no_patches)
+        else stringResource(R.string.patches_available, patchCount),
+        modifier = Modifier.clickable {
```

Comment:
Use the `enabled` parameter of `clickable` to disable clicking if `patchCount == 0`